containers/Contact/SNSBox.tsx

```diff
@@ -0,0 +1,109 @@
+import styled from 'styled-components';
+
+import { Gmail, Kakao } from '../../public';
+import { media } from '../../styles/theme';
+
+export default function SNSBox() {
+  const snsList = [
+    {
+      name: 'gmail',
+      content: 'depromeet@gmail.com',
+      logo: <Gmail />,
+      onClick: () =>
+        window.open('mailto:depromeet@gmail.com?subject=[홈페이지 문의] '),
+    },
+    {
+      name: 'kakao plus friend',
+      content: 'depromeet',
+      logo: <Kakao />,
+      onClick: () => window.open('http://pf.kakao.com/_xoxmcxed/chat'),
+    },
+  ];
```

Comment:
```suggestion
  {
      name: 'instagram',
      content: '@depromeet',
      logo: <Instagram />,
      onClick: () => window.open('https://www.instagram.com/depromeet/?hl=ko'),
    },
  ];
```

instagram 추가 하는 코드 첨부합니다!!

Reply:
감사합니다 :) 반영했습니다~~

[관련 커밋](https://github.com/depromeet/www.depromeet.com/pull/58/commits/6c2229d98a5b23b02f36f391016cd74ba4a15996)